.cspell.json

```diff
@@ -125,6 +125,8 @@
         "Dustilock",
         "Dxml",
         "Dynatrace",
+        "eadme",
```

Comment:
Can cspell use something more complete than part of a regex, since it might be a typo we'd want to catch. 
The original error comes from the regex of the last example in the descriptor:

```yaml
    examples:
      - lychee --format detailed --no-progress README.md info.txt test.html
      - lychee --format detailed --no-progress README.md
      - lychee --format detailed --no-progress test.html info.txt
      - lychee --format detailed --no-progress --offline path/to/directory
      - lychee --format detailed --no-progress https://raw.githubusercontent.com/lycheeverse/lychee/master/README.md
      - 'lychee --format detailed --no-progress "~/projects/big_project/**/README.*"'
      - 'lychee --format detailed --no-progress --glob-ignore-case --verbose "~/projects/**/[r]eadme.*"'
```

Reply:
If you have a locally running dev environment with cspell, what does it look like if you exclude instead
```json
        "[r]eadme",
```
or 

```json
        "~/projects/**/[r]eadme.*",
```

Reply:
@echoix I think that the most simple if that @DariuszPorowski removes the triggering line in the examples :)

Reply:
These examples are from their docs though.

Reply:
cspell doesn't take these partial words. So leave as it